build-system/tasks/process-github-issues.js

```diff
@@ -0,0 +1,274 @@
+            }
+            else if (issueType === 'Type: Bug' ||
+              issueType === 'Related to: Flaky Tests') {
+              // 4 is the number for Milestone 'Backlog Bugs'
+              issueNewMilestone = 4;
+              updates.push(applyMilestone(issue, issueNewMilestone));
+            }
+            else if (milestone == null) {
+              // 20 is the number for Milestone 'Pending Triage'
+              updates.push(applyMilestone(issue, issueNewMilestone));
+            }
+          }
+        }
+        else if (milestone == null) {
+          // 20 is the number for Milestone 'Pending Triage'
+          updates.push(applyMilestone(issue, issueNewMilestone));
+        }
+        else if (milestoneTitle === 'Prioritized FRs' ||
+          milestoneTitle === 'New FRs') {
+          updates.push(applyLabel(issue, 'Type: Feature Request'));
+        }
+        else if (milestoneTitle === 'Backlog Bugs' ||
+          milestoneTitle.startsWith('Sprint')) {
+          updates.push(applyLabel(issue, 'Type: Bug'));
+        }
+        // Apply default priority if no priority
+        if (hasPriority == false && milestoneTitle != 'New FRs' &&
+          milestoneTitle !== '3P Implementation' &&
+          milestoneTitle !== 'Pending Triage' && milestone != null) {
+          updates.push(applyLabel(issue, 'P2: Soon'));
+        }
+        return Promise.all(updates);
+      });
+    });
+    assert(Array.isArray(issues), `issues must be an array.`);
+    return issues;
+  });
+}
+
+/**
+ * @param {string} issue
+ * @param {number} milestoneNumber
+ * @return {!Promise<*>}
+ */
+function applyMilestone(issue, milestoneNumber) {
+  var options = extend({}, optionsMilestone);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+
+  issue.milestone = milestoneNumber;
+  return githubRequest(
+      '/issues/' + issue.number,
+      'PATCH',
+      issue.milestone, 'milestone').then(function() {
+        if (verbose) {
+          util.log(util.colors.green(
+              'Milestone applied ' + milestone + ' for #' + issue.number));
+        }
+      });
+}
+
+/**
+ * @param {string} issue
+ * @param {string} label
+ * @return {!Promise<*>}
+ */
+function applyLabel(issue, label) {
+  var options = extend({}, optionsMilestone);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+  return githubRequest(
+      '/issues/' + issue.number + '/labels',
+      'POST',
+      [label], 'label').then(function() {
+        if (verbose) {
+          util.log(util.colors.green(
+              'Label applied ' + label + ' for #' + issue.number));
+        }
+      });
+}
+
+/**
+ * @param {string} path
+ * @param {string=} opt_method
+ * @param {*} opt_data
+ * @param {string} typeRequest
+ * @return {!Promise<*>}
+ */
+function githubRequest(path, opt_method, opt_data, typeRequest) {
```

Comment:
Use a function names in the verb form, to indicate what this is doing. Like `createGithubRequest`.